feature: Print experiment graph schedules (#3)
sandialabs/firewheel_repo_base: src/firewheel_repo_base/misc/print_schedule/README.rst

.. _misc.print_schedule_mc:

###################
misc.print_schedule
###################

This MC outputs a JSON representation of the entire experiment's :ref:`vm-resource-schedule`.
If a filename is passed into the Plugin, then the JSON is written to a file; otherwise, it will print the schedule to ``stdout``.

> This JSON is useful for analyzing the expected experiment schedule and reviewing inputs to various  involve VM resources.

Reviewer: Think this may be a typo? 
```suggestion
This JSON is useful for analyzing the expected experiment schedule and reviewing inputs to various VM resources.
```

Author: ```suggestion
This JSON is useful for analyzing the expected experiment schedule and reviewing inputs for the various involved VM resources.
```

Author: Thoughts on that wording?

Reviewer: Looks good, wasn't sure if "involved" was involved in your final intention. Also, I think there may still be a double space in there.

.. _misc.print_schedule_mc:

###################
misc.print_schedule
###################

This MC outputs a JSON representation of the entire experiment's :ref:`vm-resource-schedule`.
If a filename is passed into the Plugin, then the JSON is written to a file; otherwise, it will print the schedule to ``stdout``.
This JSON is useful for analyzing the expected experiment schedule and reviewing inputs for the various involved VM resources.
This output does **NOT** include associated VM resource scripts nor binary files to reduce file size.

This MC can be placed in multiple locations of the experiment pipeline so that users can see different views of how the experiment schedule is evolving.

.. warning::

>     To ensure that the complete schedule is output, this MC should be the **last** parameter to :ref:`helper_experiment`.

Reviewer: To make sure that users understand that this warning is in addition to line 12 ("This MC can be placed in multiple locations..."), not in conflict with it, maybe we adjust the wording slightly to imply that this is case-specific?

```suggestion
    In cases where a complete schedule is the desired output, this MC must be the **last** parameter to :ref:`helper_experiment`.
```
